### PR TITLE
feat: add artwork recommendations section

### PR DESCRIPTION
**Pull request by MounirDhahri**
Resolves [ONYX-1211]

This PR adds the artworks recommendations section.

Screenshot 2024-08-21 at 12 33 27

[ONYX-1211]: https://artsyproduct.atlassian.net/browse/ONYX-1211?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/schema/v2/homeView/artworkResolvers.ts [hunk 1]
```diff
@@ -118,3 +119,8 @@ export const AuctionLotsForYouResolver: GraphQLFieldResolver<
 
   return result
 }
+
+export const RecommendedArtworksResolver: GraphQLFieldResolver<
+  any,
+  ResolverContext
+> = ArtworkRecommendations.resolve!
```
Comment (nickskalkin) [inline src/schema/v2/homeView/artworkResolvers.ts:126]: ✨

**Review by nickskalkin — APPROVED** [review]
👍

File: src/schema/v2/homeView/__tests__/HomeViewSection.test.ts [hunk 2]
```diff
@@ -2,6 +2,124 @@ import gql from "lib/gql"
...
+      const context: any = {
+        artworksLoader,
+        userID: "vortex-user-id",
+        authenticatedLoaders: {
+          vortexGraphqlLoader: vortexGraphQLAuthenticatedLoader,
+          meLoader: () => Promise.resolve({}),
+        },
+        unauthenticatedLoaders: {
+          vortexGraphqlLoader: null,
+        },
```
Comment (anandaroop) [inline src/schema/v2/homeView/__tests__/HomeViewSection.test.ts:92]: **question (non-blocking):** Any particular reason for including this?

**Review by anandaroop — APPROVED** [review]
Awesome!

**Comment by anandaroop** [comment]
@MounirDhahri just checking again if this one is stalled for any reason?

**Comment by MounirDhahri** [comment]
Nope - it should be merged already

**Pull request merged**